src/components/atoms/Dropdown.svelte

```diff
@@ -30,6 +30,15 @@
       margin-top: 0.5em;
     }
   }
+
+  @media (max-width: 350px) {
+    section {
+      margin-bottom: 0.5em;
+      select {
+        width: 100%;
+      }
+    }
+  }
```

Comment:
Wouldn't it be smarter to take this media query out and just grab it from 320px with styling?